example/index.tsx

```diff
@@ -10,9 +12,38 @@ import {
 import { CLIENT_TOKEN, ENVIRONMENT, APPLICATION_ID } from './src/ddCredentials';
 import { getTrackingConsent } from './src/utils';
 
+console.log("Starting Application with navigation library: " + navigationLib);
+if (navigationLib == "react-navigation") {
+    AppRegistry.registerRunnable(appName, async props => {
```

Comment:
Hmmm looks a bit weird that in one place we are using the `App` and in one place we are calling `startReactNativeNavigation`. I would move all this `navigation-switch` logic in the `/src/App` file, this way you can have only one place where you initialize the SDK and this `if/else` moves there.